cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/CareGaps.java

```diff
@@ -363,6 +332,31 @@ public SelectedMeasureReport measureReport() {
                     .get()
                     .getResource()));
         }
+
+        public SelectedBundle measureReportEvaluatedResourcesFound() {
+            // get resource References from the Patient Bundle
+            List<String> bundleResourceReferences = bundleReport().getEntry().stream()
+                    .map(g -> g.getResource()
+                            .getResourceType()
```

Comment:
Code style suggestion:  You could do this as a chain of map() calls each passing in a method reference